modules/ppcp-blocks/resources/js/checkout-block.js

```diff
@@ -582,6 +583,16 @@ const PayPalComponent = ( {
 		fundingSource
 	);
 
+	if ( typeof buttonAttributes !== 'undefined' ) {
+		style.height = buttonAttributes?.height
+			? Number( buttonAttributes.height )
+			: style.height;
+		style.borderRadius = buttonAttributes?.borderRadius
+			? Number( buttonAttributes.borderRadius )
+			: style.borderRadius;
+		style.color = buttonAttributes?.darkMode ? 'white' : '';
```

Comment:
We've removed `darkMode` prop in https://github.com/woocommerce/woocommerce/pull/51109. Please remove it from here too.